{"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/amplify\/internaltaskhubresult\/failure"]}],"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/InternalTaskHubResult"]]},"metadata":{"roleHeading":"Associated Type","required":true,"externalID":"s:7Amplify21InternalTaskHubResultP7FailureQa","symbolKind":"associatedtype","fragments":[{"text":"associatedtype","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Failure","kind":"identifier"},{"text":" : ","kind":"text"},{"preciseIdentifier":"s:7Amplify0A5ErrorP","kind":"typeIdentifier","text":"AmplifyError"}],"role":"symbol","modules":[{"name":"Amplify"}],"title":"Failure"},"identifier":{"url":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/InternalTaskHubResult\/Failure","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"associatedtype","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"},{"text":" : ","kind":"text"},{"identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/AmplifyError","kind":"typeIdentifier","preciseIdentifier":"s:7Amplify0A5ErrorP","text":"AmplifyError"}]}],"kind":"declarations"}],"references":{"doc://amplify-swift.Amplify/documentation/Amplify/AmplifyError":{"navigatorTitle":[{"kind":"identifier","text":"AmplifyError"}],"role":"symbol","title":"AmplifyError","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/AmplifyError","abstract":[{"type":"text","text":"Amplifyâ€™s philosophy is to expose friendly error messages to the customer that assist with debugging."},{"type":"text","text":" "},{"type":"text","text":"Therefore, failable APIs are declared to return error results with Amplify errors, which require"},{"type":"text","text":" "},{"type":"text","text":"recovery suggestions and error messages."}],"kind":"symbol","type":"topic","fragments":[{"text":"protocol","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"AmplifyError"}],"url":"\/documentation\/amplify\/amplifyerror"},"doc://amplify-swift.Amplify/documentation/Amplify/InternalTaskHubResult":{"abstract":[],"type":"topic","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/InternalTaskHubResult","title":"InternalTaskHubResult","fragments":[{"text":"protocol","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"InternalTaskHubResult"}],"url":"\/documentation\/amplify\/internaltaskhubresult","role":"symbol","navigatorTitle":[{"text":"InternalTaskHubResult","kind":"identifier"}],"kind":"symbol"},"doc://amplify-swift.Amplify/documentation/Amplify/InternalTaskHubResult/Failure":{"role":"symbol","abstract":[],"required":true,"title":"Failure","url":"\/documentation\/amplify\/internaltaskhubresult\/failure","type":"topic","fragments":[{"text":"associatedtype","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"},{"text":" : ","kind":"text"},{"preciseIdentifier":"s:7Amplify0A5ErrorP","text":"AmplifyError","kind":"typeIdentifier"}],"kind":"symbol","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify\/InternalTaskHubResult\/Failure"},"doc://amplify-swift.Amplify/documentation/Amplify":{"url":"\/documentation\/amplify","type":"topic","title":"Amplify","role":"collection","identifier":"doc:\/\/amplify-swift.Amplify\/documentation\/Amplify","kind":"symbol","abstract":[]}}}